src/BotClient.ts

```diff
@@ -159,10 +169,22 @@ export class Util {
         }
     }
 
-    static formatBridge(bid: number, bridge?: ResolvedBridge): string {
-        return bridge === undefined
-            ? `\`${bid}\`: INVALID`
-            : `\`${bridge.bridge_id}\`: \`${bridge.source_guild.name}#${bridge.source_channel.name}\` → \`${bridge.destination_guild.name}#${bridge.destination_channel.name}\` on condition \`${bridge.attribute}:${bridge.regex}\``;
+    static formatBridge(bid: number, bridge?: ResolvedBridge): { title: string; content: string } {
```

Comment:
Really not a fan of this style with multiple returns. Would you mind reworking this either into a ternary expression or in the style of
```javascript
const result = {
  title: title,
  content: "INVALID"
}

if (bridge !== undefined) {
  // ...
}

return result;
```
?

Reply:
fixed